### ISSUE TITLE
Add extra heading to action dialogue's for consistency

### ISSUE THREAD

**Issue by phptek** [silverstripe-australia/advancedworkflow#27]
For usability, accessibility and consistency's sake, an extra &lt;h2&gt; should be used for the first-section of each action's popup dialogue.

Currently one might see "Assign Users" as a heading for the second logical field-grouping, but there is no consistent equivalent above it for the first logical field-grouping above ("Action class", "Title" etc).

Suggest something like "Edit Action" or "Administer Action".
